corehq/apps/app_manager/views/modules.py

```diff
@@ -361,15 +362,27 @@ def _setup_case_property_builder(app):
 
 # Parent case selection in case list: get modules whose case type is the parent of the given module's case type
 def _get_modules_with_parent_case_type(app, module, case_property_builder, case_type_):
-        parent_types = case_property_builder.get_parent_types(case_type_)
-        modules = app.modules
-        parent_module_ids = [mod.unique_id for mod in modules
-                             if mod.case_type in parent_types]
-        return [{
-            'unique_id': mod.unique_id,
-            'name': mod.name,
-            'is_parent': mod.unique_id in parent_module_ids,
-        } for mod in app.modules if mod.case_type != case_type_ and mod.unique_id != module.unique_id]
```

Comment:
I think there might be some confusion here. @orangejenny It looks like current selection mode already allows non-parent module. Did we mean, that we display exclusively parent menus since we now have a separate 'Other' menu option? Right now per this PR, when we select parent, it displays all other modules whose case-type is not the case-type of the module (irrespective of parent/child). And when we select Other, it displays all other modules that has any case-type.

Reply:
Hmm...yeah, maybe this could be simpler. I've never really understood why the old dropdown excluded the current/child case type and then starred parent modules, instead of being truly limited to only parent modules.

Maybe we should just display a dropdown of all modules, regardless of whether you select "Parent" or "Other," and drop the `is_parent` logic that adds the stars. It would certainly simplify the code, and I doubt that the existing parent/child workflow gets that much value out of excluding the current/child modules, or out of starring the parent modules. Then the parent/other selection would only affect the expression added to the suite. What do you think? (And would it help to go to voice?)